tests/sles4sap/trento/cluster_configure.pm

```diff
@@ -0,0 +1,52 @@
+# Copyright SUSE LLC
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+# Summary: Configuration steps for qe-sap-deployment
+# Maintainer: QE-SAP <qe-sap@suse.de>, Michele Pagot <michele.pagot@suse.com>
+
+use strict;
+use warnings;
+use Mojo::Base 'publiccloud::basetest';
+use testapi;
+use base 'qesapdeployment';
```

Comment:
Having this module inheriting methods from 2 base classes worries me a bit. And from looking at the current code in the PR, I don't see a need for `qesapdeployment` being a base class. None of its methods do anything with `$self` and the only variable stored within the class is `@qesapdeployment::logs` which is used once. Perhaps with the missing methods it'll be more clear, but as things are, I would suggest changing this line to `use 'qesapdeployment';` instead.